mars-sim-core/src/main/java/com/mars_sim/core/environment/TerrainElevation.java

```diff
@@ -287,233 +263,6 @@ public double obtainRegolithCollectionRate(Coordinates loc) {
 		return site.getRegolithCollectionRate();
 	}
 
-//	/**
-//	 * Compute the RGB Topo map based Elevation.
-//	 * 
-//	 * @param location
-//	 * @return
-//	 */
-//	public static int getRGBIntElevation(Coordinates location) {	
-//		// Find hue and saturation color components at location.
-//		int color = mapDataUtil.getMapData("topo").getRGBColorInt(location.getPhi(), location.getTheta());
-//
-//		// The peak of Olympus Mons is 21,229 meters (69,649 feet) above the Mars areoid (a reference datum similar to Earth's sea level). 
-//		// The lowest point is within the Hellas Impact Crater (marked by a flag with the letter "L"). 
-//	    // The lowest point in the Hellas Impact Crater is 8,200 meters (26,902 feet) below the Mars areoid. 
-//		
-//		double height = (color - 9000) / 10_000;
-//
-//		return (int)Math.round(height);
-//	}
-	
-	/**
-	 * Returns the elevation in km at the given location, based on MEGDR's dataset.
-	 *
-	 * @param location the location in question
-	 * @return the elevation at the location (in km)
-	 */
-	public static double getColorElevation(Coordinates location) {	
-				
-		return getColorElevation(location.getPhi(), location.getTheta());	
-	}
-	
-	/**
-	 * Compute the HSB color topo map based Elevation.
-	 * 
-	 * @param phi
-	 * @param theta
- 	 * @param rho
-	 * @return elevation in km
-	 */
-	public static double getColorElevation(double phi, double theta) {	

```

Comment:
I just added the capability back in this [commit](https://github.com/mars-sim/mars-sim/commit/55e8bef6d3c858177c0e8c797bb8f97b82b88490) recently.

Topo map by definition conveys the topographically info to the viewer. 

Thus, it connects terrain to an elevation range. 

This method will eventually supplement the elevation data encoded as  a pixel color. 

However, because of jpg compression, I found that the recovered RGB colors has been degraded and no longer matches up. Therefore, the HSB values are off. I bet the tiff image (or the uncompressed jpg image) would give better result.

I will come back to this in future.

Reply:
The point is the colour terrain lookup is only used by the Console code. So we shouldn't be keeping code that is not used

Reply:
>  colour terrain lookup is only used by the Console code.

Yeah that it went through changes lately not to bother using it since more work needs to be done to parse the color of a pixel into useful elevation data.